src/nvim/change_defs.h

```diff
@@ -0,0 +1,14 @@
+#ifndef NVIM_CHANGE_DEFS_H
+#define NVIM_CHANGE_DEFS_H
```

Comment:
This file seems useless. I can't imagine a case where including this file without including `change.h` does anything.